lang/java/compiler/src/test/idl/output/simple.avpr

```diff
@@ -41,7 +35,12 @@
       "default" : "A"
     }, {
       "name" : "hash",
-      "type" : "MD5",
+      "type" : {
+        "type" : "fixed",
+        "name" : "MD5",
+        "doc" : "An MD5 hash.",
+        "size" : 16
+      },
```

Comment:
Forward references are written as inlined named types when they first occur.
This is existing, expected behaviour.